commit 4699dca74bb07472080893b96c5e13928137d97b (HEAD, no-break-in-case)
Author: Jacek Konieczny <jajcus@jajcus.net>
Date:   Wed Jun 11 10:34:38 2014 +0200

    Do not abuse 'break' inside a shell 'case'
    
    It is not C. The code would break under mksh when 'pull.ff' is set:
    
      $ git pull
      /usr/lib/git-core/git-pull[67]: break: can't break
      Already up-to-date.
    
    Signed-off-by: Jacek Konieczny <jajcus@jajcus.net>

diff --git a/git-pull.sh b/git-pull.sh
index cfc589d..18a394f 100755
--- a/git-pull.sh
+++ b/git-pull.sh
@@ -58,11 +58,9 @@ pull_ff=$(git config pull.ff)
 case "$pull_ff" in
 false)
 	no_ff=--no-ff
-	break
 	;;
 only)
 	ff_only=--ff-only
-	break
 	;;
 esac
 
